src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -65,10 +67,12 @@ ArticleTimestamp.propTypes = {
   firstPublished: number.isRequired,
   lastPublished: number.isRequired,
   popOut: bool,
+  delta: number,
 };
 
 ArticleTimestamp.defaultProps = {
   popOut: true,
+  delta: 0,
```

Comment:
Might need a second opinion on this, but again, can we rename this prop to be something like `deltaMinutes`?

Reply:
what about `minutesTolerance` or `timeDifferenceLimit`?

Reply:
I prefer `minutesTolerance` over `deltaMinutes`

Reply:
Changed to minutesTolerance :)